package.json

```diff
@@ -68,5 +68,8 @@
     "sinon": "14.0.0",
     "typescript": "4.7.4",
     "vitest": "0.19.0"
+  },
+  "dependencies": {
+    "@stablelib/sha256": "^1.0.1"
```

Comment:
Looks like this was accidentally added to the op level package.json instead of inside the auth-client’s own one